### PR TITLE
add backend for course equivalency post

### PR DESCRIPTION
**Pull request by nehageorge**

### CODE REVIEW

File: api/app.py [hunk 1]
```diff
@@ -80,6 +80,28 @@ def search_courses(query):
     course_equivalencies = course_equivalencies_join_to_dict(result)
     return jsonify(course_equivalencies)
 
+@app.route('/course/search', methods=['POST', 'GET'])
+def course_search():
+    if request.method == 'POST':
+        print("HI LOL")
```
Comment (AndreaMiranda99) [inline api/app.py]: One print statement left

File: api/app.py [hunk 2]
```diff
@@ -80,6 +80,28 @@ def search_courses(query):
...
+        uni = University.query.filter(University.name == uni).first()
+        uni_id = uni.id if uni else 54
+        uw_course = UWCourse.query.filter(UWCourse.code == uw_course).first()
+        uw_course_id = uw_course.id if uw_course else 1
+
+        db.session.add(CourseEquivalency(uwcourse_id=uw_course_id,university_id=uni_id,code="{0}: {1}".format(course_code,course_name), year_taken=year,student_program=program))
+        db.session.commit()
+        return redirect(url_for('course_search'))
+    else:
+        return jsonify("")
```
Comment (AndreaMiranda99) [inline api/app.py]: Why does the GET return nothing? if it's not needed could we just remove it or is this a placeholder for a future task?

File: src/components/XChangeButton.js [hunk 3]
```diff
@@ -7,11 +7,11 @@ const s = {
   fontStyle: "italic",
 };
 
-function XChangeButton(txt = "EMPTY", path = "", styl = s, clr = "#E0D03B") {
+function XChangeButton(txt = "EMPTY", path = "", styl = s) {
   return (
     <Link to={path}>
       <div class="button">
-        <Button sx={{ backgroundColor: clr }} style={{ width: "100%" }}>
```
Comment (aparajitaghimire) [inline src/components/XChangeButton.js]: wait can this just be kept? in case we want to customize the button colour to be something else, like gray?

Comment (nehageorge) [reply src/components/XChangeButton.js]: oh my bad i didnt mean to delete that

File: api/app.py (hunk 2 above)
Comment (nehageorge) [reply api/app.py]: yup deleting

**Review by aparajitaghimire — APPROVED** [review]
LGTM <3

**Pull request merged**